code/controllers/subsystem/ticker.dm

```diff
@@ -458,4 +461,79 @@ var/datum/subsystem/ticker/ticker
 		if(25 to INFINITY)  //No response from the next in line when a vacancy exists, remove he
 			next_in_line << "<span class='danger'>No response recieved. You have been removed from the line.</span>"
 			queued_players -= next_in_line
-			queue_delay = 0
\ No newline at end of file
+			queue_delay = 0
+
+/datum/subsystem/ticker/proc/check_maprotate(var/force = 0)
```

Comment:
The force argument for this proc looks like it got left over from the debug stuff, as its not used anywhere.